submarine-sdk/pysubmarine/tests/client/test_client.py

```diff
@@ -0,0 +1,161 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from mock import patch
+from unittest.mock import ANY
+import pytest
+from submarine.client import Client
+
+from submarine.exceptions import SubmarineException
+from submarine.proto.SubmarineServerProtocol_pb2_grpc import SubmarineServerProtocolStub
+from submarine.client.params import RoleParameter, Localization, Resource
+import submarine.proto.SubmarineServerProtocol_pb2
+
+
+def __init__(self, channel):
+    def mocked_submit(params):
+        class mocked_response:
+            application_id = 'application-123456789'
+
+        return mocked_response()
+
+    self.SubmitJob = mocked_submit
+
+
+# Submarine client parameters
+host = 'submarine'
+port = 8980
+security_disabled = True
+keytab = 'keytabPath'
+principal = 'testPrincipal'
+distribute_keytab = True
+queue = 'submarine'
+client = Client(host=host, port=port, security_disabled=security_disabled, keytab=keytab,
+                principal=principal, distribute_keytab=distribute_keytab, queue=queue)
+
+# Submarine job parameters
+name = 'test_rpc'
+docker_image = 'apache/submarine:latest'
```

Comment:
Because submarine has many types of mirrors, such as: `submairne database`, `submarine server`, `submarine-operator`, now all use naming rules such `as apache/submarine:server-VERSION`.

If you need to use a `submarine-server` image here, you need to change it to `apache/submarine:server-latest`.

Regarding the command of mirroring, we may need to discuss it in the documentation or community meetings.

Reply:
This is just an arbitrary string of docker image.
Actually, we won't run this docker images in the submarine cluster. 
I used mock_submit, so we won't pass these parameters to RPC server